src/importers.js

```diff
@@ -0,0 +1,17 @@
+/*
+Provides the async `import()` function to source code that needs it,
+without having it transpiled down to commonjs `require()` by TypeScript.
+When we drop Node 10 support, we'll stop transpiling to commonjs and remove this.
+ */
```

Comment:
We dropped Node 10 support, so, is the following still required?

Reply:
Ah good point, I need to update this comment.

We still need to support the commonjs path unfortunately because of transpilers that people use via `--require-module` - for example with TypeScript, ts-node has [only early experimental support](https://github.com/TypeStrong/ts-node/issues/1007) for handling `import()` (this is why the features in cucumber-js itself didn't work when I tried this).

So I think it's safest to still default to `require()` for now and have `import()` be opt-in via the `--esm` flag.

Reply:
Alternatively, we could use `import()` for loading all user code _except_ for requiring support files where `--require-module` is set (meaning `ts-node` or `babel` is in play and thus we'll fall back to `require`).

I think that's better actually, I'll run it up and see what happens.